defaults/main.yml

```diff
@@ -7,6 +7,7 @@
 docker_apt_key_url: "https://download.docker.com/linux/debian/gpg"
 docker_apt_keyrings_dir: "/etc/apt/keyrings"
 docker_apt_uri: "https://download.docker.com/linux/debian"
+docker_cron_image_prune: false
```

Comment:
I'd prefer a comment explaining this option, since it is also not listed in the README.

Reply:
Maybe this is a good opportunity to tackle #10 first?! 😁

Reply:
Added a comment and force-pushed.